airflow/airflow.py

```diff
@@ -41,31 +41,6 @@ def get_dag_data(self, instance, tags):
             "{}.dags_active.paused".format(self.SOURCE_TYPE_NAME), count - active_and_not_paused_dags, tags=tags
         )
 
-    def get_task_data(self, instance, tags):
-        """
-        See https://github.com/apache/airflow/blob/1.10.0/airflow/utils/state.py#L49 for available
-        task states
-        """
-        engine = sqlalchemy.create_engine(instance[AIRFLOW_SQL_ALCHEMY_CONN_KEY])
-        states = [
-            "success",
-            "failed",
-            "upstream_failed",
-            "skipped",
-            "running",
-            "queued",
-        ]
-        for state in states:
-            [count] = engine.execute(
-                """
-                select count(*) from task_instance inner join dag on task_instance.dag_id = dag.dag_id
-                where dag.is_active = true and state = '{}'
-            """.format(
-                    state
-                )
-            ).fetchone()
```

Comment:
basically replaced by `executor.queued_tasks` and `executor.running_Tasks`, 
https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/metrics.html